docs/docs/dev_docs/cli/cli-commands.md

```diff
@@ -0,0 +1,96 @@
+---
+title: CLI Commands
+---
+
+Here you will find a reference to the commands available in the Aztec CLI.
+
+## Installation
+
+This command will install the Aztec CLI.
+
+```bash
+npm install -g @aztec/cli
+```
+
+## Creating Accounts
+
+The first thing we want to do is create a couple of accounts. We will use the `create-account` command which will generate a new private key for us, register the account on the sandbox, and deploy a simple account contract which [uses a single key for privacy and authentication](../../concepts/foundation/accounts/keys.md):
+
+#include_code create-account yarn-project/end-to-end/src/cli_docs_sandbox.test.ts bash
+
+Once the account is set up, the CLI returns the resulting address, its privacy key, and partial address. You can read more about these [here](../../concepts/foundation/accounts/keys.md#addresses-partial-addresses-and-public-keys).
+
+Save the Address and Private key as environment variables. We will be using them later.
+
+```bash
+export ADDRESS=<Address printed when you run the command>
+export PRIVATE_KEY=<Private key printed when you run the command>
+```
+
+Alternatively, we can also manually generate a private key and use it for creating the account, either via a `-k` option or by setting the `PRIVATE_KEY` environment variable.
+
+#include_code create-account-from-private-key yarn-project/end-to-end/src/cli_docs_sandbox.test.ts bash
```

Comment:
These snippets include command outputs, not just the input that should be copied. The output should be removed from the copy-able section to make it easier to copy-paste.

First raised in this issue: https://github.com/AztecProtocol/aztec-packages/issues/1785

An incomplete solution opened in this Pr: https://github.com/AztecProtocol/aztec-packages/pull/2983

![image](https://github.com/AztecProtocol/aztec-packages/assets/18372439/8e463237-50fc-44cc-ade6-28612d6a192d)